app/(main)/league/all/page.test.tsx

```diff
@@ -21,7 +21,7 @@ jest.mock('@/api/apiFunctions', () => ({
 }));
 
 describe('Leagues Component', () => {
-  const mockUseDataStore = useDataStore as jest.Mock;
+  const mockUseDataStore = useDataStore as unknown as jest.Mock;
```

Comment:
Why is this unknown?

Reply:
The test was failing otherwise and I updated this line as per the typescript error:

![Code - Insiders page test tsx — gridiron-survivor — Gridiron Survivor-000012](https://github.com/user-attachments/assets/060bf9ce-223f-4c91-957f-4e8e88750c22)

Reply:
That doesn't make any sense because useDataStore was not touched at all for this codebase.

Reply:
Was this addressed @ryandotfurrer?